src/styles/index.scss

```
@@ -1,3 +1,144 @@
-body {
+.page {
+  scroll-behavior: smooth;
+  background: #fff;
+  font-family: Roboto, sans-serif;
+
+  &_body {
+    margin: 0;
+  }
+}
+
```

```suggestion
body {
  margin: 0;
}
```
Consider removing default browser margins

---

What do you mean? I can't understand what is wrong here